zcutil/fetch-params.sh

```diff
@@ -86,30 +93,10 @@ $README_PATH
 EOF
     fi
 
-    mkdir -p "$REGTEST_DIR"
-
-    fetch_params "$REGTEST_PKEY_URL" "$REGTEST_DIR/$REGTEST_PKEY_NAME"
-    fetch_params "$REGTEST_VKEY_URL" "$REGTEST_DIR/$REGTEST_VKEY_NAME"
-
     cd "$PARAMS_DIR"
 
-    # Now verify their hashes:
-    echo 'Verifying parameter file integrity via sha256sum...'
-    shasum -a 256 --check <<EOF
-226913bbdc48b70834f8e044d194ddb61c8e15329f67cdc6014f4e5ac11a82ab  regtest/$REGTEST_PKEY_NAME
-4c151c562fce2cdee55ac0a0f8bd9454eb69e6a0db9a8443b58b770ec29b37f5  regtest/$REGTEST_VKEY_NAME
-EOF
-    # Check the exit code of the shasum command:
-    CHECKSUM_RESULT=$?
-    if [ $CHECKSUM_RESULT -eq 0 ]; then
-        echo 'Updating testnet3 symlinks to regtest parameters.'
-        mkdir -p "$TESTNET3_DIR"
-        ln -sf "../regtest/$REGTEST_PKEY_NAME" "$TESTNET3_DIR/$REGTEST_PKEY_NAME"
-        ln -sf "../regtest/$REGTEST_VKEY_NAME" "$TESTNET3_DIR/$REGTEST_VKEY_NAME"
-    else
-       exit 1
-    fi
-
+    fetch_params "$BETA2_PKEY_URL" "$PARAMS_DIR/$BETA2_PKEY_NAME" "cca9887becf803c8ca801bc9da8fcba4f5fb3ba13af9d17e8603021a150cb4b7"
```

Comment:
I'd put these in variables up next to `PKEY_NAME` etc, again to reduce the size of future diffs.